mainapp/models.py

```diff
@@ -109,11 +114,20 @@ def summarise(self):
             out += "\nKit Requirements :\n {}".format(self.detailkit_util)
         if(len(self.needothers.strip()) != 0):
             out += "\nOther Needs :\n {}".format(self.needothers)
+        if self.additional_phone_numbers:
+            out += "\nAdditional phone numbers :\n {}".format("\n".join(self.additional_phone_numbers))
         return out
 
     def __str__(self):
         return self.get_district_display() + ' ' + self.location
 
+    def save(self, *args, **kwargs):
+        if self.additional_phone_numbers:
+            self.additional_phone_numbers = [item.strip() for item in self.additional_phone_numbers[0].split(' ')
+                                             if item is not '' and re.match("^\+?1?\d{9,15}$", item) is not None]
```

Comment:
You can use `if not item` instead of `if item is not ''`

Reply:
I wanted it to be explicit check for '' since if there are multiple spaces in between two numbers then we get '' after split.